### ISSUE TITLE
Placeholder for list field type

### ISSUE THREAD

**Issue by Polm90** [joomdev/JD-Simple-Contact-Form#16]
Would be possible to add a placeholder on a list field type?
Thank you!

**Comment by actuallyakash** [comment]
@Polm90 HTML Select tag doesn't provide functionality for placeholder texts.

Although, there are some hacks around to implement the placeholder functionality in which you need to modify the module's code.

**Comment by Polm90** [comment]
Yes, I know the html doesn't provide this functionality natively.
However, it's heavily required in web design, so it could be usefull if the module provides this feature, without the need to manually modify the module's code everytime :) :)

**Comment by actuallyakash** [comment]
Hey @Polm90 

Let me discuss this with the team and I'll update you when and if we're going to implement this in the module.

Thanks.

**Comment by airaketa** [comment]
Any updates on this one?
